python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -111,7 +111,8 @@ def test_categorical_accessor_initialization2(data):
         dsr.cat
 
 
-@xfail_dask_expr("TODO: Unexplained dask-expr failure")
+# TODO: Remove this once we are pinned to dask>=2024.5.0
+@xfail_dask_expr("Requires: https://github.com/dask/dask/pull/11059")
```

Comment:
Wonder if the `lt_version` param of this marker should also account for the dask-core version, since the dask-expr doesn't have a super established release cycle yet?

That way, in addition to leaving this TODO we could also do something like `lt_version=2024.5.0` to make sure that things fail loudly here once that dask-core version becomes available.

Reply:
Yeah, I was thinking the same thing. I was actually going to submit a dedicated PR to revise the `xfail_dask_expr` utility, but might as well do it here :)

Reply:
Okay - Thanks again for the suggestion. The `xfail_dask_expr`/`skip_dask_expr` utilities have been updated.